physics/PBL/SATMEDMF/satmedmfvdifq.meta

```diff
@@ -256,6 +256,30 @@
   type = real
   kind = kind_phys
   intent = in
+[def_1]
```

Comment:
@mkavulich The variables added to this scheme are all new and their standard names should be reviewed. Can you help with this? I will put my suggestions here, but please chime in whether you agree/not.